apps/community/src/app/my/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { MyInfoCardList } from '~/components/my/my-info-card-list';
+import { PageHeader } from '~/components/page-header';
+import { getMyProfile } from './remotes';
+
+export const dynamic = 'force-dynamic';
+
+export default async function MyPage() {
+  const { data } = await getMyProfile();
+
+  const userDetails = [
+    { title: '이름', value: data.name },
+    { title: '학번', value: data.id },
+    { title: '구분', value: data.role },
+    { title: '전공', value: data.major },
+  ];
+
+  const userEditableDetails = [
+    { title: '전화번호', value: data.phone },
+    { title: '이메일', value: data.email },
+  ];
+
+  return (
+    <>
+      <PageHeader
+        title="회원 정보"
+        description="등록한 회원 정보를 확인할 수 있어요."
+      />
+      <MyInfoCardList
+        userDetails={userDetails}
+        userEditableDetails={userEditableDetails}
+      />
```

Comment:
해당 부분은 details와 editableDetails를 분리하여 관리하는 것이 좋다고 생각이 드는데, 다른 분들의 의견이 궁금합니다!

Reply:
```suggestion
      <MyInfoCard
        userDetails={userDetails}
      />
      <MyInfoEditalbeCard
        userEditableDetails={userEditableDetails}
      />
```
이런 방식으로 컴포넌트를 분리하는게 좋아보인다는건가요 ??

Reply:
제 개인적인 생각으로는 MyInfoCardList로 감싸는 것이 아니라 해당 코드를 그대로 드러내는 것이 낫다고 생각이 됩니다!

```
     <MyInfoCard title="내 프로필" layout="default">
        {userDetails.map((detail) => (
          <MyInfoCard.Field
            key={detail.title}
            title={detail.title}
            value={detail.value}
          />
        ))}
      </MyInfoCard>

      <MyInfoCard title="기본 정보" layout="singleColumn">
        {userEditableDetails.map((detail) => (
          <MyInfoCard.EditableField
            key={detail.title}
            title={detail.title}
            value={detail.value}
            onSave={(value) => handleSave(detail.title, value)}
          />
        ))}
      </MyInfoCard>
```

Reply:
MyPage는 async/await를 포함하는 서버 컴포넌트이고, MyInfoCardList컴포넌트는 상태관리와 이벤트 핸들링을 포함한 클라이언트 컴포넌트라서 분리를 해줬습니다.
말씀하신 방법대로 코드를 드러내는건 무리라고 생각하고, 다른 접근방법이 있다면 말씀해주세요 !!

Reply:
제가 놓친 부분이 있었네요😂 그럼 일단 넘어가는 걸로 할까요?